spec/services/sync/fetch_profile_spec.rb

```diff
@@ -133,5 +133,41 @@
         expect(Profile.find_by(nickname: profile_name)[:url]).to eq(profile_url)
       end
     end
+
+    describe 'profile from Github: doesn\'t exist, profile from DB: exists' do
+      let(:location)  { create(:location) }
+      let(:profile) { create(:profile, nickname: profile_name, location: location) }
+
+      before do
+        profile_call_fetch_profile = instance_double(
+          Github::ProfileConsumer,
+          call: {},
+          code: 404,
+          body: nil
+        )
+        allow(Github::ProfileConsumer).to receive(:new).and_return(profile_call_fetch_profile)
+
+        profile
+      end
+
```

Comment:
In my opinion, it's a smell of how the spec is badly structured when you have to call the variable alone just for the sole purpose of loading the `let` declaration. I see 2 potential solutions here:
1. (Not recommented) Use `let!` instead of `let` to enforce the variable definition instead of lazy loading it.
2. For this case, do you really need to have a helper variable called `profile`? I see you are not using anywhere else in the spec, only here. So, why not just create the profile here on the `before` hook instead of inside the `let`? So it would look like this:
```suggestion
      let(:location)  { create(:location) }

      before do
        profile_call_fetch_profile = instance_double(
          Github::ProfileConsumer,
          call: {},
          code: 404,
          body: nil
        )
        allow(Github::ProfileConsumer).to receive(:new).and_return(profile_call_fetch_profile)

        create(:profile, nickname: profile_name, location: location)
      end
```